### PR TITLE
[Coverity] fix coverity

### PR DESCRIPTION
**Pull request by andrei-cv**
### Details:
 - *fix coverity*

### Tickets:
 - *CVS-112753*
 - *CVS-112763*
 - *CVS-112760*

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp [hunk 1]
```diff
@@ -224,9 +224,9 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
     }
 
     kernel_arguments_data get_arguments_impl(const typed_primitive_inst<PType>& instance) const override {
-        for (size_t k = 0; k < _kernels.size(); ++k) {
+        if (_kernels.size()) {
```
Comment (vladimir-paramuzov) [inline src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp:227]: Why?

Comment (andrei-cv) [reply src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp:227]: The loop makes only one step, if _kernels.size()>0. Return in line 235.

Comment (vladimir-paramuzov) [reply src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp:227]: @e-ddykim Is it expected behavior?

Comment (andrei-cv) [reply src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp:227]: It seems that yes.

**Review by vladimir-paramuzov — APPROVED** [review]

**Pull request merged**